internal/bminventory/inventory_test.go

```diff
@@ -3132,21 +3134,34 @@ var _ = Describe("cluster", func() {
 						ClusterID: clusterID,
 						ClusterUpdateParams: &models.V2ClusterUpdateParams{
 							UserManagedNetworking: swag.Bool(true),
+							APIVip:                swag.String("10.35.20.9"),
 							IngressVip:            swag.String("10.35.20.10"),
 						},
 					})
-					verifyApiErrorString(reply, http.StatusBadRequest, "Ingress VIP cannot be set with User Managed Networking")
+					verifyApiErrorString(reply, http.StatusBadRequest, "Ingress VIPs cannot be set with User Managed Networking")
 				})
 
-				It("Fail with API VIP", func() {
+				It("Fail with API and Ingress VIPs", func() {
+					By("Fail VIPs with User Managed Networking")
 					reply := bm.V2UpdateCluster(ctx, installer.V2UpdateClusterParams{
 						ClusterID: clusterID,
 						ClusterUpdateParams: &models.V2ClusterUpdateParams{
 							UserManagedNetworking: swag.Bool(true),
 							APIVip:                swag.String("10.35.20.10"),
+							IngressVip:            swag.String("10.35.30.10"),
 						},
 					})
-					verifyApiErrorString(reply, http.StatusBadRequest, "API VIP cannot be set with User Managed Networking")
+					verifyApiErrorString(reply, http.StatusBadRequest, "Ingress VIPs cannot be set with User Managed Networking")
+
+					By("Fail API VIP with no Ingress VIP")
+					reply = bm.V2UpdateCluster(ctx, installer.V2UpdateClusterParams{
+						ClusterID: clusterID,
+						ClusterUpdateParams: &models.V2ClusterUpdateParams{
+							UserManagedNetworking: swag.Bool(true),
+							APIVip:                swag.String("10.35.20.10"),
+						},
+					})
+					verifyApiErrorString(reply, http.StatusBadRequest, "configuration must include the same number of api_vips (got 1) and ingress_vips (got 0)")
```

Comment:
Could we write those errors in English instead of param names? I.e. instead of
```
configuration must include the same number of api_vips (got 1) and ingress_vips (got 0)
```
would have
```
configuration must include the same number of API VIPs (got 1) and ingress VIPs (got 0)
```

I'm suggesting this because we also have KubeAPI where the field is **not** called `api_vip` but `apiVIPs` so there may be a confusion

Reply:
done